src/pages/PeoplePage.tsx

```
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Loader } from '../components/Loader';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { PeopleTable } from '../components/PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState<string | null>(null);
+  const { slug } = useParams();
+
+  useEffect(() => {
+    const fetchPeople = async () => {
+      try {
+        const data = await getPeople();
+
+        setPeople(data);
+      } catch {
+        setError('Failed to fetch people');
```

The `catch` block swallows the thrown error because it uses `catch { ... }`. Capture the error parameter so you can use its message for debugging and/or show a more informative error to users. For example: `catch (err) { console.error(err); setError(err?.message || 'Failed to fetch people'); }`. This will make it easier to diagnose fetch failures.